src/include/btmem.h

```diff
@@ -232,8 +232,8 @@ struct __wt_ovfl_reuse {
  * We also configure a larger than default internal page size to accommodate for larger history
  * store keys. We do that to reduce the chances of having to create overflow keys on the page.
  */
-#ifdef HAVE_BUILTIN_EXTENSION_SNAPPY
-#define WT_HS_COMPRESSOR "snappy"
+#ifdef HAVE_BUILTIN_EXTENSION_ZSTD
+#define WT_HS_COMPRESSOR "zstd"
```

Comment:
Just to point that only the newer databases will use the `zstd`. All the existing versions still point to `snappy`. Is this what expected or the existing versions also want to switch?

Reply:
Yes, that is what is expected. we will not change the existing one's, only the new one's.